code/parse/sexp.cpp

```diff
@@ -32015,6 +32054,8 @@ int query_operator_argument_type(int op, int argnum)
 			else
 				return OPF_VARIABLE_NAME;
 
+		case OP_SET_FRIENDLY_DAMAGE_CAPS:
+			return OPF_POSITIVE;
```

Comment:
This should be `OPF_NUMBER` otherwise FREDders would not be able to pass -1 to turn off the cap